### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by nanle771** [zhang-tao-whu/DVIS#12]
How can I replicate similar results on only one GPU? What parameters need to be modified?
Can I display only the segmentation mask (without categories and scores) when running the demo.py for visualization?
Thank you!

**Comment by zhang-tao-whu** [comment]
Perhaps you could decrease the learning rate and batch size while increasing the number of iterations accordingly. For example, ```BASE_LR: 0.0000125, IMS_PER_BATCH: 1, MAX_ITER: 160000, STEPS: (112000,)```.

**Comment by zhang-tao-whu** [comment]
If you want to hide the class and score, please modify the draw_instance_predictions function on line 50 of [visualizer.py](https://github.com/zhang-tao-whu/DVIS/blob/main/demo_video/visualizer.py).
